ui/README.md

```diff
@@ -2,6 +2,18 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 13.2.3.
 
+## Updating the UI
+
+The `dist/ui` directory is mounted inside the harbourbridge binary using the `go embed` library.
+After making any changes to the UI, perform the following steps to update Harbourbridge with the UI changes:
+
+1. Run `ng build` inside the `ui/` directory. This places the generated angular artifacts in the `ui/dist/ui` directory.
+2. Run `make build` from the root directory to generate the harbourbridge binary.
+3. Run `./harbourbridge web` to run the web UI.
+4. Navigate to `http://localhost:4200/` to access the UI.
```

Comment:
Will this continue to be 4200 with the port change ?
If not, lets pass through the code to see any other references to the port and remove it

Reply:
Thanks for catching this! Forgot to update it, will change.